src/main/java/org/sonarlint/intellij/actions/UpdateAction.java

```diff
@@ -19,55 +19,76 @@
  */
 package org.sonarlint.intellij.actions;
 
-import com.intellij.openapi.actionSystem.AnAction;
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.application.ApplicationManager;
 import com.intellij.openapi.progress.ProgressIndicator;
 import com.intellij.openapi.progress.ProgressManager;
 import com.intellij.openapi.progress.Task;
+import com.intellij.openapi.project.Project;
 import com.intellij.openapi.ui.Messages;
 import org.jetbrains.annotations.NotNull;
-import org.sonarlint.intellij.console.SonarLintConsole;
-import org.sonarlint.intellij.inspection.SonarQubeRunnerFacade;
+import org.sonarlint.intellij.analysis.SonarLintStatus;
+import org.sonarlint.intellij.ui.SonarLintConsole;
+import org.sonarlint.intellij.analysis.SonarQubeRunnerFacade;
 
-public class UpdateAction extends AnAction {
+import javax.swing.*;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Explicitly import the specific classes needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2208)